From 2f1b9a29b0540e4c6b8064076192a7b2a3544dc2 Mon Sep 17 00:00:00 2001
From: Andrea Arcangeli <aarcange@redhat.com>
Date: Thu, 5 Jan 2012 19:31:09 +0100
Subject: [PATCH] KSM: add manpage entry for -redhat-disable-KSM

RH-Author: Andrea Arcangeli <aarcange@redhat.com>
Message-id: <1325791869-14304-2-git-send-email-aarcange@redhat.com>
Patchwork-id: 36250
O-Subject: [PATCH] KSM: add manpage entry for -redhat-disable-KSM
Bugzilla: 719269
RH-Acked-by: Marcelo Tosatti <mtosatti@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>

Moving the definition up inside a @table with the other options
documented in the manpage, the last options at the end of the file
aren't documented in the manpage either (including -mem-prealloc and
-mem-path etc...).

Signed-off-by: Andrea Arcangeli <aarcange@redhat.com>
---
 qemu-options.hx |   10 +++++++---
 1 files changed, 7 insertions(+), 3 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qemu-options.hx |   10 +++++++---
 1 files changed, 7 insertions(+), 3 deletions(-)

diff --git a/qemu-options.hx b/qemu-options.hx
index dd08256..195b091 100644
--- a/qemu-options.hx
+++ b/qemu-options.hx
@@ -301,6 +301,13 @@ a suffix of ``M'' or ``G'' can be used to signify a value in megabytes or
 gigabytes respectively.
 ETEXI
 
+DEF("redhat-disable-KSM", 0, QEMU_OPTION_disable_KSM,
+    "-redhat-disable-KSM  disable KSM on guest physical memory\n")
+STEXI
+@item -redhat-disable-KSM
+Disable KSM on the virtual RAM.
+ETEXI
+
 DEF("k", HAS_ARG, QEMU_OPTION_k,
     "-k language     use keyboard layout (for example 'fr' for French)\n")
 STEXI
@@ -2112,6 +2119,3 @@ DEF("fake-machine", 0, QEMU_OPTION_fake_machine,
     "-fake-machine        create a fake machine incapable of running guest code\n"
     "                     mimimal resource use, use for scalability testing")
 #endif
-
-DEF("redhat-disable-KSM", 0, QEMU_OPTION_disable_KSM,
-    "-redhat-disable-KSM  disable KSM on guest physical memory\n")
-- 
1.7.7.4

